README.md

```diff
@@ -330,753 +277,454 @@ Rails.application.routes.draw do
   # Defines the root path route ("/")
   # root "articles#index"
 
-  get "/", to: "welcome#index"
-
   get "/tasks", to: "tasks#index"
 end
 ```
 
-Thinking back to the welcome page that we created, what are our next steps to get this route to work properly? In order to display an error free view, we will need to add a `TasksController`, an `index` action in that controller, and an `index.html.erb` file for that controller action.
+This line that we added is telling our application that anytime we receive an HTTP GET request for the URI '/tasks', we should perform the index action within our tasks controller.
+
+Go back to your browser and refresh the page - you should now be seeing an error telling you that you have a `Routing Error`; specifically, that you have an `uninitialized constant TasksController`. This is a good thing; at this point, we have told our application which controller action to perform when this request is received, but we haven't created that controller (or the action) yet. So, let's go do that.
+
+### Adding the Controller
 
-First, add a new controller to your controllers directory called `tasks_controller.rb`
+Open your `app/controllers` directory and create a file inside called `tasks_controller.rb`. In that file, add the following code:
 
-**app/controllers/tasks_controllers.rb**
+**app/controllers/tasks_controller.rb**
 
 ```ruby
 class TasksController < ApplicationController
   def index
-    @tasks = ["Task 1", "Task 2", "Task 3"]
+    render json: {
+      example_key: {
+        example_nested_key: "It worked!"
+      },
+      example_array_key: [
+        "Array item 1", "Array item 2"
+      ]
+    }
```

Comment:
Maybe add a note here: Does this remind you of anything? (hint: Nested collections in mod 1)